recipe/run_test.py

```diff
@@ -0,0 +1,43 @@
+import os
+from subprocess import check_call
+import sys
+
+# openmpi env, before imoprting mpi4py
+os.environ["OMPI_MCA_plm"] = "isolated"
+os.environ["OMPI_MCA_btl_vader_single_copy_mechanism"] = "none"
+os.environ["OMPI_MCA_rmaps_base_oversubscribe"] = "yes"
+os.environ["OMPI_ALLOW_RUN_AS_ROOT"] = "1"
```

Comment:
No, no, this is not going to work. These variables are read by mpiexec, before spawning the actual executable.